### ISSUE TITLE
__init__() got an unexpected keyword argument 'extra_files'

### ISSUE THREAD

**Issue by simonw** [rsalmaso/django-serve#1]
I tried running this but it failed to start:
```
% ./manage.py serve
[2021-09-19 23:38:20 +0000] [9869] [INFO] Starting gunicorn 20.1.0
[2021-09-19 23:38:20 +0000] [9869] [INFO] Listening at: http://127.0.0.1:8000 (9869)
[2021-09-19 23:38:20 +0000] [9869] [INFO] Using worker: sync
[2021-09-19 23:38:20 +0000] [9870] [INFO] Booting worker with pid: 9870
[2021-09-19 23:38:20 +0000] [9870] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/workers/base.py", line 131, in init_process
    self.reloader = reloader_cls(extra_files=self.cfg.reload_extra_files,
TypeError: __init__() got an unexpected keyword argument 'extra_files'
[2021-09-19 23:38:20 +0000] [9870] [INFO] Worker exiting (pid: 9870)
[2021-09-19 23:38:20 +0000] [9871] [INFO] Booting worker with pid: 9871
[2021-09-19 23:38:20 +0000] [9871] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/workers/base.py", line 131, in init_process
    self.reloader = reloader_cls(extra_files=self.cfg.reload_extra_files,
TypeError: __init__() got an unexpected keyword argument 'extra_files'
[2021-09-19 23:38:20 +0000] [9871] [INFO] Worker exiting (pid: 9871)
[2021-09-19 23:38:20 +0000] [9872] [INFO] Booting worker with pid: 9872
[2021-09-19 23:38:20 +0000] [9872] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/Users/simon/.local/share/virtualenvs/simonwillisonblog-izSE3ZQu/lib/python3.9/site-packages/gunicorn/workers/base.py", line 131, in init_process
    self.reloader = reloader_cls(extra_files=self.cfg.reload_extra_files,
TypeError: __init__() got an unexpected keyword argument 'extra_files'
[2021-09-19 23:38:20 +0000] [9872] [INFO] Worker exiting (pid: 9872)
[2021-09-19 23:38:20 +0000] [9869] [WARNING] Worker with pid 9872 was terminated due to signal 15
[2021-09-19 23:38:20 +0000] [9869] [WARNING] Worker with pid 9871 was terminated due to signal 15
[2021-09-19 23:38:20 +0000] [9869] [INFO] Shutting down: Master
[2021-09-19 23:38:20 +0000] [9869] [INFO] Reason: Worker failed to boot.
```

**Comment by glasnt** [comment]
I got this issue too, and found that applying the pending PR patch from https://github.com/benoitc/gunicorn/pull/2353 gets me a working error message: 

```
ImportError: You must have the inotify module installed to use the inotify reloader
```

I'm running macOS Big Sur, so it's probably related to that (inotify being a linux thing, fswatch being a macos thing). 

Given this is designed to help django run in production and production environments aren't normally macOS-based OSs, this is a me problem :)

**Comment by rsalmaso** [comment]
Hi, thanks for this issue (so I know that someone find this package useful :smile_cat: )
Yes, it's a macos problem because `django-serve` currently requires inotify.
I'm working to remove this hard requirements.

**Comment by rsalmaso** [comment]
I just released 0.2.0, and now
1) install inotify only on linux
2) can choose to use the reloader or not (default yes)
3) works on macos (for now only in poll mode)

Thanks
